extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -162,19 +162,57 @@
 		@include column-count(4);
 		display: none;
 
-		label {
-			display: inline-block;
-			position: relative;
-			width: 100%;
+		span {
+			display: block;
+			margin-left: 32px;
 		}
 
-		input {
-			position: absolute;
-		}
+		.checkbox-styled {
+			float: left;
+			position: relative;
+			width: 18px;
+
+			label {
+				border: 1px solid $wds-color-light-gray;
+				border-radius: 1px;
+				cursor: pointer;
+				height: 18px;
+				left: 0;
+				position: absolute;
+				top: 0;
+				width: 18px;
+
+				&:after {
+					background-color: transparent;
+					border: 2px solid $wds-color-white;
+					border-top: none;
+					border-right: none;
+					content: '';
+					height: 4px;
+					left: 4px;
+					opacity: 0;
+					position: absolute;
+					top: 4px;
+					transform: rotate(-45deg) translate3d(0,0,0);
```

Comment:
I saw recently `transform` property being browser-prefixed. However, http://caniuse.com/#search=transform shows that prefixes are not needed for browsers we support. We're dropping the mixin? I'd say yes, but wanted to be sure. :)